terraform/scripts/restart_dockerd_with_tls.sh

```diff
@@ -5,20 +5,20 @@ export DEBIAN_FRONTEND=noninteractive
 # Make docker daemon only accept connections with trusted certificate
 
 TARGET_DIR=/lib/systemd/system/docker.service.d
-mkdir -p "$TARGET_DIR"
+sudo mkdir -p "$TARGET_DIR"
 
 # override default ExecStart
-cat > "$TARGET_DIR/local.conf" << EOF
+sudo bash -c 'cat << EOF > $TARGET_DIR/local.conf
```

Comment:
`shellcheck restart_docker_with_tls.sh`
 returns the following error:
>   SC1004: This backslash+linefeed is literal. Break outside single quotes if you just want to break the line.

maybe use double quotes
```suggestion
sudo bash -c "cat << EOF > $TARGET_DIR/local.conf
```